i3pystatus/core/util.py

```diff
@@ -373,7 +373,7 @@ def __new__(cls):
         try:
             socket.create_connection(cls.address, 1).close()
             return True
-        except OSError:
+        except (OSError, socket.gaierror):
```

Comment:
Since Python 3.3 gaierror is also a subclass of OSError, so this should only affect 3.2 and earlier.